cocotb/handle.py

```diff
@@ -216,10 +216,15 @@ def _sub_handle_key(self, name):
         return name.split(".")[-1]
 
     def _getAttributeNames(self):
-        """Permits IPython tab completion to work."""
+        """Permits older IPython tab completion to work."""
         self._discover_all()
```

Comment:
You can remove this line now, since `dir` handles it

Reply:
Actually, I'd just remove this function altogether - `__dir__` has been supported by ipython from its very first git commit:

https://github.com/takluyver/ipython/blame/11c30a1d83c24ccad09f8f8063a5f8ba36c3137d/IPython/genutils.py#L1924-L1954

Reply:
Removed.